docs/CONFIG.md

```diff
@@ -1213,6 +1213,40 @@ ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
 ListenAddresses is the addresses the peer will listen to on the network in `host:port` form as accepted by `net.Listen()`,
 but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
 
+## Capabilities.RateLimit
+```toml
+[Capabilities.RateLimit]
```

Comment:
I'm not 100% convinced by these limits; they still seem quite high to me

Taking the baseline we expect to have to handle for ST3, how about:
- GlobalRPS = 200
- GlobalBurst = 200
- PerSenderRPS = 50 // Higher than 10 because I think SAs will be using a single MCMS account to register workflows; might be something to check with Lei
- PerSenderBurst = 50

I don't think this needs to block the PR though; we can take this as a follow up